### DISCUSSION TITLE
flea/tick.

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1237] [General]
Hi, I have a 3 year old intact male lab.

He has hyperkeratosis on his paws, managed.

for over two years he never had fleas, I do worm count every three months (so far no worms)

However he got fleas from a holiday cottage and we have not been able to clear them, cleaning, indorex, and otc treatment every month.

not massive amounts, only seen on him.

im moving towards a vet prescription, and im frankly terrified by bravecto or medicines of that nature. I was going to go for advocate drop on but I have read really bad reactions.

im being offered a generic advocate or simpatica, as i have expressed I am concerned about the other types.

He is showing potentially FAD and I have tried all the natural remedies, neem, essential oils.

any recommendations or advice. thanks

**Comment by hernessteve** [comment]
Bravecto isn’t the only tablet option. Mine have a monthly tablet, credelio, but only when they are likely to be in places where there might be fleas or ticks. Like you, I’ve only had dogs with fleas once, when my sister gave me her sofa and she had a cat. Controlling the environment is as important if not more important than treating the dog.

**Comment by hernessteve** [comment]
What is your opinion about Nexgard for Labs, guys? I’m really curious to hear your thoughts because I’ve been considering it for my dog. Labs tend to be active and adventurous, which makes them more exposed to fleas and ticks, so finding an effective solution is super important. My friends recently told me that it’s now possible to get [Nexgard Without Vet Prescription](https://www.worldpetexpress.net/blog/nexgard-without-vet-prescription ), which is a huge plus in my book. It saves time and effort, especially when you’re juggling a busy schedule and don’t want the extra hassle of a vet visit just to get preventative medication. What are some other advantages you’ve noticed with Nexgard?
